src/spider/worker/worker.cpp

```diff
@@ -219,6 +232,92 @@ parse_outputs(spider::core::Task const& task, std::vector<msgpack::sbuffer> cons
     return outputs;
 }
 
+/**
+ * Handles the result of a task execution. Parse the task outputs and submit them to the storage.
+ *
+ * @param storage_factory Factory for creating storage connections
+ * @param metadata_store Metadata storage for submitting results
+ * @param instance Task instance that was executed
+ * @param task The task that was executed
+ * @param executor The executor that ran the task
+ * @return true if results were successfully handled, false if any errors occurred
```

Comment:
```suggestion
 * @param storage_factory Factory for creating storage connections.
 * @param metadata_store Metadata storage for submitting results.
 * @param instance Task instance that was executed.
 * @param task The task that was executed.
 * @param executor The executor that ran the task.
 * @return true if results were successfully handled, false if any errors occurred.
```
try to be consistent about using or not using periods (I'm adding them because all the previous doc strings had them)